### PR TITLE
various enhancements

### PR DESCRIPTION
**Pull request by laskasn**

### CODE REVIEW

File: src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentRepresentation.java [hunk 1]
```diff
@@ -0,0 +1,46 @@
+package org.keycloak.plugins.groups.representations;
+
+import org.keycloak.representations.idm.GroupRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+
+import java.util.Collection;
+
+public class GroupEnrollmentRepresentation {
```
Comment (cgeorgilakis) [inline src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentRepresentation.java:8]: Add default constructor

File: src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentStateRepresentation.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
+package org.keycloak.plugins.groups.representations;
+
+import org.keycloak.plugins.groups.jpa.entities.GroupEnrollmentEntity;
+import org.keycloak.representations.idm.GroupRepresentation;
+import org.keycloak.representations.idm.UserRepresentation;
+
+import javax.persistence.Column;
+import java.util.Collection;
+
+public class GroupEnrollmentStateRepresentation {
```
Comment (cgeorgilakis) [inline src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentStateRepresentation.java:10]: Add default constructor

File: src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentRepresentation.java (hunk 1 above)
Comment (laskasn) [reply src/main/java/org/keycloak/plugins/groups/representations/GroupEnrollmentRepresentation.java:8]: done

**Review by cgeorgilakis — APPROVED** [review]

**Comment by laskasn** [comment]
thanks Kostas!

**Pull request merged**